agent/api/eni.go

```diff
@@ -86,7 +94,22 @@ func ENIFromACS(acsenis []*ecsacs.ElasticNetworkInterface) []*ENI {
 		})
 	}
 
-	return enis
+	return enis, nil
+}
+
+func ValidateTaskENI(acsenis []*ecsacs.ElasticNetworkInterface) error {
+	// Only one eni should be associated with the task
+	// Only one ipv4 should be associated with the eni
+	// ONly one ipv6 should be associated with the eni
+	if len(acsenis) != 1 {
+		return errors.Errorf("enifromacs: task contains %d eni,expected one", len(acsenis))
+	} else if len(acsenis[0].Ipv4Addresses) != 1 {
+		return errors.Errorf("eni contains %d ipv4, expected one", len(acsenis[0].Ipv4Addresses))
```

Comment:
Please stick to the format of "error context: error message" for error messages. Example: `"eni message validation: more than one ipv4 addresses in the message (%d)"`